pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -609,10 +614,21 @@ func handleInitVerifiedObjects(ctx *verifierContext) {
 func verifyImageStatusFromImageFile(imageFileName string,
 	size int64, pathname string) *types.VerifyImageStatus {
 
+	// filename might have two parts, separated by '.': digest and base64(mediaType)
+	var (
+		mediaType string
+		digest    string
+	)
+	parts := strings.Split(imageFileName, ".")
+	digest = parts[0]
+	if len(parts) == 2 {
+		mediaType = url.PathUnescape(parts[1])
```

Comment:
Should we verify the string here? 
Or use just the int value, as we do here?
Maybe we can use just the int value that corresponds to the type?
Like we do here?
https://github.com/lf-edge/eve/blob/5a06e7997aa9e85c94ab539eaed3b09f7d1721d4/pkg/pillar/cmd/volumemgr/initialvolumestatus.go#L141

Reply:
I really prefer not to. This is a media-type, it could mean lots of things, including those of which perhaps EVE is unaware. These should be passed through as is.